rpc/swap.go

```diff
@@ -54,11 +56,11 @@ type GetPastRequest struct {
 
 // GetPastResponse ...
 type GetPastResponse struct {
-	Provided       types.ProvidesCoin `json:"provided"`
-	ProvidedAmount float64            `json:"providedAmount"`
-	ReceivedAmount float64            `json:"receivedAmount"`
-	ExchangeRate   types.ExchangeRate `json:"exchangeRate"`
-	Status         string             `json:"status"`
+	Provided       coins.ProvidesCoin  `json:"provided"`
+	ProvidedAmount *apd.Decimal        `json:"providedAmount"`
+	ReceivedAmount *apd.Decimal        `json:"expectedAmount"`
```

Comment:
```suggestion
	ReceivedAmount *apd.Decimal        `json:"receivedAmount"`
```

since the swap was in the past, so the amount was received already?